### PR TITLE
Removed v2 upgrade

### PR DESCRIPTION
**Pull request by sampocs**
**Summary**
* Removed v2 upgrade (since this will be a chain reset) but kept boilerplat

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -44,5 +44,5 @@ vue/*
 .terraform
 *.pem
 
-.vscode/*
+.vscode
```
Comment (asalzmann) [inline .gitignore:47]: Out of curiosity, what's the diff here?

**Review by asalzmann — APPROVED** [review]
lgtm!

tip: you can request @Stride-Labs/core-chain-eng as a reviewer to ping everyone now!

File: .gitignore (hunk 1 above)
Comment (sampocs) [reply .gitignore:47]: New one will look recursively into each directory, old one only looks at the root level

Comment (asalzmann) [reply .gitignore:47]: nice! sorry, think you may have mentioned this on an older PR and I missed it

Comment (sampocs) [reply .gitignore:47]: No worries! It’s pretty unintuitive, I still don’t really understand why it works like that

**Pull request merged**